main.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+$(document).ready(function(){
+
+    //cheese pizza calculations here
+
+    let cheeseCalorieCounter = +$('.calorie_num').val();
+    let cheeseFatCounter = +$('.fat_num').val();
+    let cheeseCarbCounter = +$('.carb_num').val();
+    let cheeseProteinCounter = +$('.protein_num').val();
+
+
+
+
+    $('#original_cheese_pizza').click(function() {
+
+        //bug on hover effect, doesn't apply css except on when you scroll down then scroll back up?????
+        //then gets stuck on hover effect
```

Comment:
I haven't worked in jQuery in some time, so I'm not quite sure how to fix this bug. But perhaps you can just do your hover effect in CSS? Something like the following:
```
#original_cheese_pizza:hover {
  opacity: 0.5;
}
```